openlibrary/templates/lists/feed_updates.html

```diff
@@ -31,7 +31,7 @@
                     <name>$c.author.displayname</name>
                     <uri>$request.home$c.author.key</uri>
                 $else:
-                    <name>Anonymous</name>
+                    <name>$_("Anonymous")</name>
```

Comment:
This isn't actually html ; it's I think an XML response, so we likely don't want to i18n this.